### DISCUSSION TITLE
Session Storage and Local Storage

### DISCUSSION THREAD

**Discussion by gourav2110** [dotnetcore/BootstrapBlazor#3509] [Q&A]
@ArgoZhang 

Please add Session Storage and Local Storage (Client-Side)

**Comment by ArgoZhang** [comment]
@gourav2110 hi, please do not `@` `LORDofDOOM` `alexinea` `cairabbit` `Oceania2018` `EdiWang` `Ahoo-Wang`. They are not project members. thanks

what does Session/Local Storage?

https://learn.microsoft.com/en-us/aspnet/core/blazor/state-management?view=aspnetcore-8.0&pivots=server#save-and-load-data-within-a-component&wt.mc_id=DT-MVP-5004174

- ProtectedLocalStorage
- ProtectedSessionStorage

**Reply by gourav2110** [reply]
@ArgoZhang 

Browser storage
